api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -987,6 +997,12 @@ public static final List<GlobalProperty> CORE_GLOBAL_PROPERTIES() {
 		                "3",
 		                "An integer which, if specified, would determine the maximum number of encounters to display on the encounter tab of the patient dashboard."));
 		
+		props.add(new GlobalProperty(GP_ENCOUNTER_EDITABLE_DURATION, "",
+		        "The length of time in minutes after encounter creation during which the encounter is allowed to be edited. By default, there is no limit."));
```

Comment:
What do you think about including, in the description, an example or two, of such time values?

Reply:
Sure, good idea. Done.